.github/CONTRIBUTING.md

```diff
@@ -25,49 +25,52 @@ In VDP, a **pipeline** is a DAG (Directed Acyclic Graph) consisting of multiple
 
 ```mermaid
 flowchart LR
-    s[Trigger] --> c1[OpenAI Connector]
-    c1 --> c2[Stability AI Connector]
-    c1 --> c3[MySQL Connector]
+    s[Trigger] --> c1[OpenAI Component]
+    c1 --> c2[Stability AI Component]
+    c1 --> c3[MySQL Component]
     c1 --> e[Response]
     c2 --> e
 ```
 
 ### Component
 
 There are different types of component:
-- **connector**
-  - Queries, processes or transmits the ingested data to a service or app.
-  - Users need to configure their connectors (e.g. by providing an API token to
-    a remote service).
+- **ai**
+  - Transforms unstructured data into formats that are easy to interpret and analyze, thereby facilitating the extraction of valuable insights.
+- **data**
+  - Establishes connections with remote data sources, such as IoT devices (e.g., IP cameras), cloud storage services (e.g., GCP Cloud Storage, AWS S3), data warehouses, or vector databases (e.g., Pinecone).
+- **application**
+  - Integrates various 3rd-party application services
 - **operator**
   - Performs data injection and manipulation.
 - **iterator**
   - Takes an array and executes an operation (defined by a set of nested
     components) on each of its elements.
```

Comment:
```suggestion
- **AI**
  - Transforms unstructured data into formats that are easy to interpret and analyze, thereby facilitating the extraction of valuable insights.
- **Data**
  - Establishes connections with remote data sources, such as IoT devices (e.g., IP cameras), cloud storage services (e.g., GCP Cloud Storage, AWS S3), data warehouses, or vector databases (e.g., Pinecone).
- **Application**
  - Integrates various 3rd-party application services
- **Operator**
  - Performs data injection and manipulation.
- **Iterator**
  - Takes an array and executes an operation (defined by a set of nested
    components) on each of its elements.
```

Reply:
Maybe we can integrate these short descriptions as a first paragraph in the following sub-sections?

Reply:
Indeed, let me think how to merge them.

Reply:
I remove those short descriptions since they are duplicated..